.github/workflows/node-publish.yml

```diff
@@ -13,18 +13,15 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Setup Node
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v2
       with:
         node-version: '14'
+        registry-url: 'https://registry.npmjs.org'
 
-    - name: Install Dependencies
+    - name: Setup Environment
       run: npm install
-
-    - name: Build package
-      run: npm run build
-
+      
     - name: Publish Package
       env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        NPM_TOKEN: ${{ secrets.SEMANTIC_RELEASE_NPM_TOKEN }}
-      run: npm run semantic-release
+        NODE_AUTH_TOKEN: ${{ secrets.SEMANTIC_RELEASE_NPM_TOKEN }}
+      run: npm publish
```

Comment:
we usually follow `semantic-release` for our JS packages as done here: https://github.com/edx/paragon/blob/master/.github/workflows/release.yml

Reply:
I firstly tried to do it with semantic release but after adding semantic release package Node_publish workflow and  even CI workflow too would start to fail on this repo with a message : `Could not install from "node_modules/karma-jasmine-jquery/bower-installer@git:/github.com/bessdsv/bower-installer.git#temp" as it does not contain a package.json file.` 
I couldn't produce it locally on my machine and then I tried this approach.
For more Insights you can have a look at this [PR](https://github.com/edx/edx-custom-a11y-rules/pull/27)